client.go

```diff
@@ -226,6 +226,10 @@ func (c *Client) Reconnect() error {
 	return nil
 }
 
+func (c *Client) Runtime() string {
```

Comment:
This needs a documentation string, we are still working on getting that fixed in CI :)

Reply:
happy to do a follow up PR as soon as this merges. it's a struggle to get the CI jobs to go green as you can see from history :)